src/terms.jl

```diff
@@ -580,12 +580,13 @@ hasresponse(t::FormulaTerm) =
     term(x)
 
 Wrap argument in an appropriate `AbstractTerm` type: `Symbol`s become `Term`s,
```

Comment:
```suggestion
Wrap argument in an appropriate `AbstractTerm` type: `Symbol`s and `AbstractString`s become `Term`s,
```
(Needs a line break but you get the idea.)